internal/proto/controller/api/resources/authmethods/v1/auth_method.proto

```diff
@@ -74,77 +74,77 @@ message AuthMethod {
 }
 
 // The attributes of an OIDC typed auth method.
 message OidcAuthMethodAttributes {
   // Output only. The state of the auth method. Will be "inactive",
   // "active-private", or "active-public".
-  string state = 10 [json_name = "state"];
+  string state = 10 [json_name = "state"];  // @gotags: `class:"sensitive"`
 
   // The issuer URL. Boundary expects only the schema, host, and port and will
   // strip off ".well-known/openid-configuration" if present. This will be
   // used for configuration discovery as well as for validation of the "iss"
   // claim.
-  google.protobuf.StringValue issuer = 20 [json_name = "issuer", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.issuer" that: "Issuer" }];
+  google.protobuf.StringValue issuer = 20 [json_name = "issuer", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.issuer" that: "Issuer" }];  // @gotags: `class:"sensitive"`
 
   // An OAuth 2.0 Client Identifier valid at the Authorization Server.
   google.protobuf.StringValue client_id = 30
-      [json_name = "client_id", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_id" that: "ClientId" }];
+      [json_name = "client_id", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_id" that: "ClientId" }];  // @gotags: `class:"sensitive"`
 
   // Input only. The client's secret.
   google.protobuf.StringValue client_secret = 40
-      [json_name = "client_secret", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_secret" that: "ClientSecret" }];
+      [json_name = "client_secret", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.client_secret" that: "ClientSecret" }];  // @gotags: `class:"secret"`
 
   // Output only. The HMAC'd value of the clients secret to indicate whether
   // the client secret has changed.
-  string client_secret_hmac = 50 [json_name = "client_secret_hmac"];
+  string client_secret_hmac = 50 [json_name = "client_secret_hmac"];  // @gotags: `class:"secret"`
 
   // The OIDC "max_age" parameter sent to the Authorization Server indicating
   // a maximum acceptable time in seconds since the user's last authentication
   // before requiring the user to reauthenticate. 0 indicates an immediate
   // need to reauthenticate.
-  google.protobuf.UInt32Value max_age = 60 [json_name = "max_age", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.max_age" that: "MaxAge" }];
+  google.protobuf.UInt32Value max_age = 60 [json_name = "max_age", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = { this: "attributes.max_age" that: "MaxAge" }];  // @gotags: `class:"sensitive"`
```

Comment:
Hum... I guess sensitive is okay here.

Reply:
Turns out this doesn't actually do anything: https://github.com/hashicorp/boundary/pull/1995/files#diff-a325f09243ab1eb7aedffd7e7be649202909a1c9b20a8c88e09435b8d475820cR91